app/src/main/java/com/cornellappdev/transit/ui/components/AddFavoritesSearchSheet.kt

```diff
@@ -142,7 +142,7 @@ fun AddFavoritesSearchSheet(
                                 MenuItem(
                                     Icons.Filled.Place,
                                     label = it.name,
-                                    sublabel = if (it.type == Type.busStop) "Bus Stop" else it.detail.toString(),
+                                    sublabel = if (it.type == PlaceType.BUS_STOP) "Bus Stop" else it.detail.toString(),
```

Comment:
Sorry to keep adding comments! But this seems like a line of code you're repeating a lot: `it.type == PlaceType.BUS_STOP) "Bus Stop" else it.detail.toString()`

It sounds to me like this is an actual property of a Place! Perhaps you'd want this as a property called `sublabel`. But this isn't something that backend supplies; it's something that you compute from the data class parameters exclusively... sounds like you should make a constant property!

```kotlin
/**
 * Data class representing an apple place or a TCAT stop
 */
@Serializable
data class Place(
    @Json(name = "lat") var latitude: Double,
    @Json(name = "long") var longitude: Double,
    @Json(name = "name") var name: String,
    @Json(name = "detail") val detail: String?,
    @Json(name = "type") var type: PlaceType
) {

   // NEW CODE!
   val subLabel = if (type == PlaceType.BUS_STOP) "Bus Stop" else detail.toString()
}
```

(That should work...)

Now, everywhere you need a bus stop's sublabel, you just reference this property! And no longer do you have to look back and think--how's that computed again?

Reply:
(Plz replace prev occurrences! And if you want to call it something else be my guest! The main idea is to make this constant field thingy, whatever you wanna call it.)

Reply:
Also, just for safety, you might want to use the following syntax instead (it's somethign like this):

```
val subLabel
    get() = if (type == PlaceType.BUS_STOP) "Bus Stop" else detail.toString()
```

Because some of your fields aren't constant; this will recompute the sublabel on demand whenever it's queried for instead of computing it once and storing it forever. The original runs the risk of the sublabel still saying bus stop even if you mutate `type` to an `APPLE_PLACE`.